frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -2592,3 +2593,32 @@ An array with a pair of state value and it's set function.
 
 
 
+---
+
+## `useModal`
+
+### Summary [DEPRECATED]
```

Comment:
@opayne1 FYI, we're deprecating a hook from the console plugin API. We should look at release noting this. Let us know what we need to do.

Reply:
Not sure how I missed this comment, but I will add to the release notes. Thanks!